packages/contracts/evm-contracts/contracts/orderbook/OrderbookDex.sol

```diff
@@ -52,6 +56,14 @@ contract OrderbookDex is IOrderbookDex, ERC1155Holder, Ownable, ReentrancyGuard
         orderCreationFee = _orderCreationFee;
     }
 
+    /// @inheritdoc IOrderbookDex
+    function claim() external {
+        uint256 amount = balances[msg.sender];
+        delete balances[msg.sender];
+        emit BalanceClaimed(msg.sender, amount);
+        payable(msg.sender).sendValue(amount);
+    }
```

Comment:
should we only allow people to claim their own balance? It might be useful if another wallet can force-claim on behalf of somebody else

If you don't think this is useful in practice for any situation, I'm fine keeping it as-is

Reply:
I can imagine some situations where that might be useful, but since we just agreed on making the contract upgradeable, let's leave the feature for future upgrades, if the need for it is there.